.editorconfig

```diff
@@ -10,3 +10,6 @@ trim_trailing_whitespace = true
 
 [*.yml]
 indent_size = 2
+
+[Makefile]
+indent_style = tab
```

Comment:
This file here to make it easier to follow indentation conventions.

And yes, tabs. Makefiles are special.